tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -164,11 +172,23 @@ def get_image_tensors(image_paths, info, batch_sizes):
             if process_with_original_shapes:
                 logger.info(f'Image will be processed with original shape - {image.shape[:-1]}')
             elif info.layout.has_name('H') and info.layout.has_name('W'):
-                new_im_size = (widthes[i % num_shapes], heights[i % num_shapes])
+                new_im_size = (widths[i % num_shapes], heights[i % num_shapes])
                 if image.shape[:-1] != new_im_size:
                     logger.warning(f"Image is resized from ({image.shape[:-1]}) to ({new_im_size})")
                     image = cv2.resize(image, new_im_size)
 
+            if info.scale.size or info.mean.size:
+                blue, green, red = cv2.split(image)
+                if info.mean.size:
+                    red = np.subtract(red, info.mean[0])
+                    green = np.subtract(green, info.mean[1])
+                    blue = np.subtract(blue, info.mean[2])
+                if info.scale.size:
+                    red = np.divide(red, info.scale[0])
+                    green = np.divide(green, info.scale[1])
+                    blue = np.divide(blue, info.scale[2])
+                image = cv2.merge([blue, green, red])
```

Comment:
Why was this part removed?

Reply:
It is now fused into the model https://github.com/openvinotoolkit/openvino/pull/14621